src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -277,7 +277,7 @@ Some elements of NRQL used in charts don’t make sense in the streaming context
       </td>
 
       <td>
-        The `COMPARE WITH` clause is used to compare the values for two different time ranges. This type of query is incompatible with NRQL alerting. We recommend using a [Baseline Alert Condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-baseline-alert-conditions) to dynamically detect deviations for a particular signal.
+        The `COMPARE WITH` clause is used to compare the values for two different time ranges. This type of query is incompatible with NRQL alerting. We recommend using a [Anomaly Alert Condition](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-anomaly-alert-conditions) to dynamically detect deviations for a particular signal.
```

Comment:
Did this URL actually change as a result of this sweep? If only the doc title (and not the filename) changed, I don't think this link would've changed, right?